### DISCUSSION TITLE
Viewer count for CMAF (Dash) Playback.

### DISCUSSION THREAD

**Discussion by Mohit-3196** [ant-media/Ant-Media-Server#5406] [General]
While looking for the approximate number of viewers a specific server type can handle, I came across some posts in which it is mentioned that for a particular server having enough bandwidth the number of supported HLS can have 4-5 times more viewers than WebRTC.

Since HLS has a latency of around 10 seconds, I'm considering using CMAF to play the streams for a relatively low latency of 3 to 5 seconds. So how does the viewer count change for CMAF playback as compared to WebRTC and HLS and what are the deciding factor for CMAF count? Is it the CPU or the bandwidth?

Thank you.

**Comment by burak-58** [comment]
Thank you for this question.
WebRTC is a real-time streaming protocol that works through direct connection between peers and sends frames as quick as possible. So it consumes more CPU as well as bandwidth. But CPU is major factor.

HLS and CMAF are HTTP based protocols which consumes less CPU but adds more latency while creating and chunk files. Both consume as much bandwidth as WebRTC but less CPU. So major bottleneck for these protocols are bandwidth.

You may estimate rough viewer limit by dividing the server bandwidth to the video bitrate what is played.
